onedal/common/_estimator_checks.py

```diff
@@ -0,0 +1,43 @@
+#===============================================================================
+# Copyright 2021 Intel Corporation

```

Comment:
2022*

Reply:
Repository hasn't patched yet, I guess we can merge it as is if it would be merged before copyrights updating.

Reply:
Applied as repo has already patched